public/main.bundle.js

```diff
@@ -107,19 +83,44 @@
 					}
 	});
 
-	var Alphabet = _react2.default.createClass({
-					displayName: 'Alphabet',
+	var CapitalAButton = _react2.default.createClass({
+					displayName: 'CapitalAButton',
+
+					showCapitalA: function showCapitalA() {
+									_reactDom2.default.render(_react2.default.createElement(Letter, null), document.querySelector("#letter"));
+					},
+
+					render: function render() {
+									return _react2.default.createElement(
+													'div',
+													{ id: 'capital-button', onClick: this.showCapitalA },
+													'A'
+									);
+					}
+
+	});
+
+	var ButtonChoices = _react2.default.createClass({
+					displayName: 'ButtonChoices',
+
 					render: function render() {
 									return _react2.default.createElement(
 													'div',
-													{ id: 'a' },
-													_react2.default.createElement(ButtonChoices, null),
-													_react2.default.createElement(Letter, null)
+													{ id: 'buttons' },
+													_react2.default.createElement(
+																	'div',
+																	{ id: 'button-group' },
+																	_react2.default.createElement(CapitalAButton, null)
+													)
 									);
 					}
 	});
 
-	_reactDom2.default.render(_react2.default.createElement(Alphabet, null), document.querySelector('#container'));
+	_reactDom2.default.render(_react2.default.createElement(
+					'div',
+					{ id: 'a' },
+					_react2.default.createElement(ButtonChoices, null)
+	), document.querySelector('#container'));
 
 /***/ },
 /* 2 */
```

Comment:
I think you're doing more than you give yourself credit for and taking on react is a big challenge.  I don't know enough to give react feedback but it's cool you're experimenting and getting good results!